packages/components/button/src/button.ts

```diff
@@ -64,7 +64,10 @@ export const buttonProps = buildProps({
   /**
    * @description determine whether it's a plain button
    */
-  plain: Boolean,
+  plain: {
+    type: Boolean,
+    default: undefined,
+  },
```

Comment:
Why change to undefined?

Reply:
If a Boolean type does not explicitly specify a default value, the default value is `false`. 

This will cause the global configuration in the logic of line 45 in `use-button` to never take effect.

Reply:
OK